keps/sig-network/4559-kubelet-probes/README.md

```diff
@@ -213,35 +217,25 @@ UNRESOLVED section in the Summary.
       exec probe that uses `curl` or the like to connect to that IP
       from the pod itself.)
 
-    - If we implement this by creating a new set of probe/lifecycle handler
-      types, rather than by modifying the semantics of the existing probes,
-      then it should be possible to block the old probe types via Pod
-      Security admission, so that administrators can create
-      NetworkPolicies that don't include the kubelet probe hole.
-
 - Allow dual-stack / wrong-single-stack probing, either based on
```

Comment:
We could move this into future goals and outline the fact that we should have expandable changes that don't lock us in in case in future we want to do this work